components/org.wso2.carbon.identity.api.server.action.management/org.wso2.carbon.identity.api.server.action.management.v1/src/extend/java/org/wso2/carbon/identity/api/server/action/management/v1/PreUpdatePasswordActionResponse.java

```diff
@@ -99,15 +101,17 @@ public boolean equals(java.lang.Object o) {
                 Objects.equals(this.getDescription(), actionResponse.getDescription()) &&
                 Objects.equals(this.getStatus(), actionResponse.getStatus()) &&
                 Objects.equals(this.getEndpoint(), actionResponse.getEndpoint()) &&
+                Objects.equals(this.getCreatedAt(), actionResponse.getCreatedAt()) &&
```

Comment:
Shall we add these two before the Endpoint line ? So the entire class has the same order of accessing the attributes

Reply:
Fixed with https://github.com/wso2/identity-api-server/pull/959/commits/dea9d362c7230f564eae5f36c82c30a778c7d60c